code/game/machinery/doors/airlock.dm

```diff
@@ -587,46 +587,46 @@ There are 9 wires.
 	verbs -= /obj/machinery/door/airlock/proc/try_wedge_item
 	verbs += /obj/machinery/door/airlock/proc/take_out_wedged_item
 
-/obj/machinery/door/airlock/proc/try_wedge_item(mob/living/user)
+/obj/machinery/door/airlock/proc/try_wedge_item()
 	set name = "Wedge item"
 	set category = "Object"
 	set src in view(1)
 
-	if(!user)
-		user = usr
-
-	var/obj/item/weapon/tool/T = user.get_active_hand()
+	if(!usr)
+		log_debug("Verb 'Wedge item' bugged and called without usr.")
```

Comment:
pretty sure usr is ALWAYS present, something is calling the proc after all.
The real check (if you REALLY want to use unreliable ass 'usr') would be checking if usr is mob.

Reply:
U can't present argument of mob/user in verb